aws/s3/secret-files/slack-secrets.yaml

```diff
@@ -0,0 +1,3 @@
+slack_secrets:
```

Comment:
This seems to be missing from scan results in the CLI. I've also tried the following format:
 `slack_secrets:
  - slack_secret1: "xoxb-not-a-real-bot-token"
  - slack_secret2: "xoxp-not-a-real-personal-token"`

Reply:
no need to do for Slack as its are not real secrets :)

Reply:
So these are expected to not be flagged by the CLI, so we shouldn't include them?

Reply:
up to you. In original `secrets.yaml` we did include it as an example that Radar can filter out such false positives I think...

Reply:
ahhh gotcha.